internal/protoc/downloader.go

```diff
@@ -42,18 +42,27 @@ import (
 	"go.uber.org/zap"
 )
 
+const _filePrefix = "file://"
```

Comment:
The `_` isn't used anywhere else in Prototool (or Golang, for that matter :-) ).